src/mono/mono/metadata/sre.c

```diff
@@ -4634,13 +4633,13 @@ ves_icall_ModuleBuilder_WriteToFile (MonoReflectionModuleBuilderHandle mb, HANDL
 {
 	mono_image_create_pefile (MONO_HANDLE_RAW (mb), file, error);
 }
+#endif
 
 void
 ves_icall_ModuleBuilder_build_metadata (MonoReflectionModuleBuilderHandle mb, MonoError* error)
 {
 	mono_image_build_metadata (MONO_HANDLE_RAW (mb), error);
 }
```

Comment:
This icall is not used in netcore Mono. Why do you need to enable it?

Reply:
No need. Removed